api/src/main.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import annotations
+
+import numpy as np
+import ray
+import json
+from ray import serve
+from fastapi import FastAPI
+from fastapi.responses import JSONResponse
+from fastapi.encoders import jsonable_encoder
+from models import Schedule
+from typing import Union
+from hypergraph import HyperGraph
+from lib import preprocess
+
+from models import (
+    Error,
+    InputData,
+    IsValidSchedule,
+    Schedule,
+    Success,
```

Comment:
Looks like this model isn't being used anywhere

Reply:
Just removed Success and Error types, good catch